financial-connections/src/main/java/com/stripe/android/financialconnections/features/partnerauth/PartnerAuthViewModel.kt

```diff
@@ -75,63 +75,55 @@ internal class PartnerAuthViewModel @Inject constructor(
 
     init {
         handleErrors()
-        withState {
-            if (it.activeAuthSession == null) {
-                launchBrowserIfNonOauth()
-                createAuthSession()
-            } else {
-                logger.debug("Restoring auth session ${it.activeAuthSession}")
-                restoreAuthSession()
-            }
-        }
+        launchBrowserIfNonOauth()
+        // TODO
+        withState { restoreOrCreateAuthSession() }
     }
 
-    private fun restoreAuthSession() {
-        suspend {
-            // if coming from a process kill, there should be a session
-            // re-fetch the manifest and use its active auth session instead of creating a new one
-            val sync: SynchronizeSessionResponse = getOrFetchSync()
-            val manifest: FinancialConnectionsSessionManifest = sync.manifest
-            val authSession = manifest.activeAuthSession ?: createAuthorizationSession(
-                institution = requireNotNull(manifest.activeInstitution),
-                sync = sync
-            )
-            Payload(
-                authSession = authSession,
-                institution = requireNotNull(manifest.activeInstitution),
-                isStripeDirect = manifest.isStripeDirect ?: false
-            )
-        }.execute { copy(payload = it) }
-    }
+    private fun restoreOrCreateAuthSession() = suspend {
+        // A session should have been created in the previous pane and set as the active
+        // auth session in the manifest.
+        // if coming from a process kill, we'll fetch the current manifest from network,
+        // that should contain the active auth session.
+        val sync: SynchronizeSessionResponse = getOrFetchSync()
```

Comment:
key change: The Auth Session is now created on the pane that launches the partner_auth pane, rather than within the pane itself. 

This mimics what web does, and it's also a requirement since we need to know if the session is `OAuth` or not in order to decide if the pane will be bottom sheet or full screen.